pkg/backingstore/reconciler.go

```diff
@@ -528,36 +532,18 @@ func (r *Reconciler) ReadSystemInfo() error {
 			))
 		}
 
-		const maxPoolNameLength = int(43)
-		if len(r.BackingStore.Name) > maxPoolNameLength {
-			return util.NewPersistentError("TooLongPoolName",
-				fmt.Sprintf("NooBaa BackingStore %q is in rejected phase due to too long pvpool backingstore name, max allowed is %d", r.BackingStore.Name, maxPoolNameLength))
-		}
-
 		const defaultVolumeSize = int64(20 * 1024 * 1024 * 1024) // 20Gi=20*1024^3
-		const minimalVolumeSize = int64(16 * 1024 * 1024 * 1024) // 16Gi=16*1024^3
 		var volumeSize int64
+
 		pvPool := r.BackingStore.Spec.PVPool
 		if pvPool.VolumeResources != nil {
 			qty := pvPool.VolumeResources.Requests[corev1.ResourceName(corev1.ResourceStorage)]
 			volumeSize = qty.Value()
-		}
-		if volumeSize < minimalVolumeSize {
 			if volumeSize == 0 {
```

Comment:
volume size will never be 0 because we reject it in the validation, we should decide if we want to keep the same behavior or not

Reply:
we can change the validation to allow 0 and then the reconciler will assign the default value.

Reply:
@romayalon @dannyzaken WDYT?

Reply:
I think it'll work

Reply:
any idea why we allowed 0 here before?

Reply:
yeah its weird, in the cli we forbid 0 value, only above the 16Gi minimum.

Reply:
> 

if the reconciler identifies 0 value it will assign the default size, which is 20G